src/xdocs/config_blocks.xml

```diff
@@ -867,6 +867,7 @@ for(int i = 0; i &lt; 10; value.incrementValue()); // OK
           method definitions, class definitions, constructor definitions,
           instance and static initialization blocks.
           The policy to verify is specified using the property <code> option</code>.
+          For right curly brace of expression blocks please follow issue #5945.
```

Comment:
please make it as a link (wrap by html tag) to issue

Reply:
@romani Sorry my bad. Corrected.